clusterloader2/testing/load/experimental-config.yaml

```diff
@@ -11,20 +11,25 @@
 {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
 {{$LOAD_TEST_THROUGHPUT := DefaultParam .CL2_LOAD_TEST_THROUGHPUT 10}}
-{{$BIG_GROUP_SIZE := 250}}
-{{$MEDIUM_GROUP_SIZE := 30}}
-{{$SMALL_GROUP_SIZE := 5}}
-{{$SMALL_STATEFUL_SETS_PER_NAMESPACE := 1}}
-{{$MEDIUM_STATEFUL_SETS_PER_NAMESPACE := 1}}
+{{$BIG_GROUP_SIZE := DefaultParam .BIG_GROUP_SIZE 250}}
```

Comment:
I actually think we should go the other way around.
Instead of changing this experimental config, change the main one (i.e. config.yaml).

Reply:
Yes, we are discussing this on chat right now

Reply:
I made diff and apply changes in this PR: https://github.com/kubernetes/perf-tests/pull/1485